.github/workflows/ci.yml

```diff
@@ -40,6 +40,22 @@ jobs:
           flags: ${{ runner.os }}
           token: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: SonarCloud Scan
+        uses: sonarsource/sonarcloud-github-action@v1.8
+        with:
+          args: >
+            -Dsonar.organization=calyptia
+            -Dsonar.projectKey=${{ env.CI_REPOSITORY_NAME_SLUG }}
+            -Dsonar.sources=.
+            -Dsonar.exclusions=**/*_test.go
+            -Dsonar.tests=.
+            -Dsonar.test.inclusions=**/*_test.go
+            -Dsonar.go.coverage.reportPaths=coverage.out
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@patrick-stephens should be this `GITHUB_TOKEN` like this one? 
https://github.com/calyptia/cli/blob/cecf9fcad38dc7f242e32ba30c6853d4ab2056bf/.github/workflows/ci.yml#L245